docs/site/content/en/docs/Tasks/manage-reports/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Manage Reports
+date: 2024-02-21
+description: How to manage Reports and Report configurations in Horreum
+categories: [Tutorial]
+weight: 3
+---
+
+## Background
+
+Creation of Reports in Horreum is straightforward enough but deletion can be not obvious. A Report in Horreum can be updated or saved as a seperate versions (configurations). A useful procedure when modifying an existing Report that functions correctly.
+
+## Report Deletion
+
+{{% imgproc total-reports-on-Report-list-page Fit "460x528" %}}
+Select Report configurations
+{{% /imgproc %}}
+
+To delete an existing report configuration select the folder icon in the Total reports column on the Reports list view. Each instance of a configuration will have a red coloured button named Delete.
```

Comment:
The "report configuration" is the list of reports defined at `http://example.com/reports`  a "report"  is an instance of a report configuration populate with data. This sentence is referring to a "report configuration" instead of a "report"

Reply:
This really needs to be in `Core Concepts` as well. I'll update this section.

Reply:
sounds good

Reply:
Having worked on Reports for a while and modifying the configuration of the Report and saved the new instance the intuitive train of thought is to call those instances a "Report configuration". The distinguishing factor is the configuration itself.
 To call the "Report" in the list a "Report configuration" as you have explained seems unintuitive. Are you adamant it should be described this way ?

Reply:
The UI refers to the list a "Report configuration" 

![image](https://github.com/Hyperfoil/Horreum/assets/959822/727eba7c-2970-4681-b665-6e5dd124e02c)

and the report instances as reports;

![image](https://github.com/Hyperfoil/Horreum/assets/959822/769cf6bf-57fb-44b3-8467-5f73749ebd83)


If we start calling them something else in the docs, then that will be confusing for users

Reply:
so it does.